canvasapi/user.py

```diff
@@ -236,12 +236,15 @@ def get_assignments(self, course_id, **kwargs):
             :class:`canvasapi.assignment.Assignment`
         """
         from canvasapi.assignment import Assignment
+        from canvasapi.course import Course
+
+        course_id = obj_or_id(course, "id", (Course,))
```

Comment:
Second param (currently "id") should be the name of the variable. In this case, `'course'`.